packages/wasm-utils/src/utxo.rs

```diff
@@ -97,19 +97,19 @@ impl JsUtxo {
 
 	pub fn get_amount(&self) -> Vec<u8> {
 		match &self.inner {
-			JsUtxoInner::Bn254(bn254_utxo) => bn254_utxo.amount.into_repr().to_bytes_le(),
+			JsUtxoInner::Bn254(bn254_utxo) => bn254_utxo.amount.into_repr().to_bytes_be(),
 		}
 	}
 
 	pub fn get_blinding(&self) -> Vec<u8> {
 		match &self.inner {
-			JsUtxoInner::Bn254(bn254_utxo) => bn254_utxo.blinding.into_repr().to_bytes_le(),
+			JsUtxoInner::Bn254(bn254_utxo) => bn254_utxo.blinding.into_repr().to_bytes_be(),
 		}
 	}
 
 	pub fn get_secret_key(&self) -> Vec<u8> {
 		match &self.inner {
-			JsUtxoInner::Bn254(bn254_utxo) => bn254_utxo.keypair.secret_key.into_repr().to_bytes_le(),
+			JsUtxoInner::Bn254(bn254_utxo) => bn254_utxo.keypair.secret_key.unwrap().into_repr().to_bytes_be(),
```

Comment:
@nepoche I think this function should now return an option. These unwraps are not good, especially when you're making the secret key optional.